pandas/plotting/_core.py

```diff
@@ -2711,7 +2766,8 @@ def line(self, x=None, y=None, **kwds):
         x, y : label or position, optional
             Coordinates for each point.
         `**kwds` : optional
-            Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
+            Additional keyword arguments are documented in
+ :py:meth:`pandas.DataFrame.plot`.
```

Comment:
here went something wrong with the indentation (and some below as well)

Reply:
Sorry, fixing...